projects/1.18-fabric/assets/auto-workstations-fabric/autoworkstations/lang/zh_cn.json

```
@@ -0,0 +1,37 @@
+{
+  "itemGroup.autoworkstations.items": "自动化工作站",
+  "block.autoworkstations.iron_auto_crafting_table": "铁质自动工作台",

```

```suggestion
  "block.autoworkstations.iron_auto_crafting_table": "铁自动工作台",
```
```suggestion
  "block.autoworkstations.iron_auto_crafting_table": "铁制自动工作台",
```
两者选其一吧
若需更改则所有铁、金、下界合金开头的物品都需更改